packages/forms/src/Form.js

```diff
@@ -211,4 +212,7 @@ Form.defaultProps = {
 
 Form.displayName = 'TalendForm';
 
+export {
+	I18N_DOMAIN,
+};
```

Comment:
Why are you importing/exporting the i18n domain ?

Reply:
I exporting the i18n domain to be able to reuse it when i create the i18n config in my application and, so, to avoid this to hard-coded the domain.
e.g:

```
.init({
		resources: {
			en: {
				[I18N_DOMAIN_COMPONENTS]: TRANSLATIONS_COMPONENTS_EN,
				[I18N_DOMAIN_FORMS]: TRANSLATIONS_FORMS_EN,
			},
			fr: {
				[I18N_DOMAIN_COMPONENTS]: TRANSLATIONS_COMPONENTS_FR,
				[I18N_DOMAIN_FORMS]: TRANSLATIONS_FORMS_FR,
			},
		},
	});
```